lib/fluent/plugin/slack_client.rb

```diff
@@ -114,7 +114,8 @@ def post_message(params = {}, opts = {})
       private
 
       def encode_body(params = {})
-        params.to_json
+        # https://api.slack.com/docs/formatting
+        params.to_json.gsub(/&/, '&amp;').gsub(/</, '&lt;').gsub(/>/, '&gt;')
```

Comment:
CGI.escapeHTML is available > http://docs.ruby-lang.org/ja/search/query:CGI.escapeHTML/version:2.2.0/#entry-0